src/main/java/net/minecraftforge/client/ForgeHooksClient.java

```diff
@@ -61,10 +65,7 @@
 import net.minecraft.inventory.EquipmentSlotType;
 import net.minecraft.item.ItemStack;
 import net.minecraft.item.crafting.RecipeManager;
-import net.minecraft.util.Direction;
-import net.minecraft.util.Hand;
-import net.minecraft.util.MovementInput;
-import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.*;
```

Comment:
Forge *tries* to stay away from wildcard imports (you have another being added lower down in this file as well)